packages/cdktf-cli/test/get/util.ts

```diff
@@ -1,31 +1,29 @@
 import { promises as fs } from 'fs';
-import { withTempDir } from "../../lib/util";
-import { jsiiCompile } from "../../lib/get/jsii"
+import { mkdtemp } from "../../lib/util";
 import { Language, GetBase } from "../../lib/get/base";
+import * as path from 'path';
 
 export function expectImportMatchSnapshot(target: string, fn: () => GetBase) {
   jest.setTimeout(60_000);
 
   test(target, async () => {
-    await withTempDir('get-cdktf', async () => {
-      const [ name ] = target.split('@');
-      const workdir = '.';
+    await mkdtemp(async workdir => {
       const importer = fn();
+      const jsiiPath = path.join(workdir, '.jsii');
 
       await importer.get({
         codeMakerOutput: workdir,
+        outputJsii: jsiiPath,
         targetLanguage: Language.TYPESCRIPT,
         targetNames: [target]
       });
 
-      await jsiiCompile(workdir, {
-        stdout: true,
-        name: name,
-        main: name,
-        providerPath: './providers/aws/index'
-      });
+      const manifest = JSON.parse(await fs.readFile(jsiiPath, 'utf-8'));
+
+      // patch cdktf version in manifest because it's not stable
+      manifest.dependencies.cdktf = '999.999.999';
```

Comment:
Could you elaborate on this number?

Reply:
It could be anything. That value is just what cdk8s uses.

The idea is that the dependency version doesn't provide any meaningful information to the snapshot and so should be effectively ignored.